Sources/MapboxNavigation/LanesView.swift

```diff
@@ -10,6 +10,8 @@ open class LanesView: UIView, NavigationComponent {
     
     public var isCurrentlyVisible: Bool = false
     
+    var isTopBannerDisplayingSteps: Bool = false
```

Comment:
`LanesView` should not know about other views. I think we should remove this boolean check and make checks whether to hide `linesView` and other banners in `TopBannerViewController`.

Reply:
This boolean check was added to make sure that the view was updated without showing the view itself -- the main cause of the transparent gap. Agreed that `LanesView` shouldn't have access to other views, but this variable was needed since adding a new parameter to `update(for:animated:duration:completion:)`, even an optional with a default, would be backwards-incompatible. We can bubble things up to the `TopBannerViewController` level, but I am a bit concerned about leaving the call to`show(animated:duration:completion:)` as is.